src/api/reservation.ts

```diff
@@ -0,0 +1,31 @@
+import axios from 'axios'
+
+import { Reservation, ReservationFormValues } from '../types'
+
+const EMPTY_RESERVATION: Reservation = {
+  id: 1,
+  monday: [],
+  tuesday: [],
+  wednesday: [],
+  thursday: [],
+  friday: [],
+  saturday: [],
+  sunday: [],
+}
+
+axios.defaults.baseURL = 'http://localhost:3000'
+
+export const fetchReservation = async () => {
+  const response = await axios.get(`/reservations/1`)
+  return response.data as Reservation
```

Comment:
It's beneficial to assert type for data from backend, as the type is then enforced wherever we use it in components.